test/integration/targets/win_unzip/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+# NOTE: The win_scheduled_task module only works on Win2012+
+
+#- name: Test Windows capabilities
```

Comment:
I left it commented because I wasn't sure this module was supported for all platforms when I started doing the tests.